ClientApp/src/app/services/labels/label.service.ts

```diff
@@ -0,0 +1,43 @@
+import { HttpClient } from '@angular/common/http';
+import { Injectable } from '@angular/core';
+import { environment } from 'src/environments/environment';
+import { LabelData } from 'src/types/label'
+
+@Injectable({
+  providedIn: 'root'
+})
+export class LabelService {
+
+  constructor(private http: HttpClient) { }
+
+  label: LabelData = { label: "", color: "#ff0000"}
```

Comment:
I'm seeing that it's the responsibility of other components to be in charge of this variable, which seems kinda silly in my opinion? this service should be doing all that abstraction instead of the component, I see that this service ends up being pretty bare instead. This is more of a subjective one I'll admit, but if the service is going to be this bare, you may as well just make the http calls directly from the component, which defeats the whole point of creating these services, to abstract that away from the components